docs/java-rest/high-level/snapshot/put_repository.asciidoc

```diff
@@ -0,0 +1,139 @@
+[[java-rest-high-snapshot-put-repository]]
+=== Snapshot Put Repository API
```

Comment:
in the docs as well, can you rename put repository to create repository. We leave requests and responses as they are as we reuse them from the transport client, but for the rest we should follow the spec naming like other language clients already do.

Reply:
I went ahead and changed it in all the headings and links (eg, `[[java-rest-high-snapshot-create-repository]]`)